flake.nix

```diff
@@ -5,7 +5,7 @@
       url = "github:numtide/flake-utils";
     };
     horizon-platform = {
-      url = "git+https://gitlab.homotopic.tech/horizon/horizon-platform";
+      url = "github:blackheaven/horizon-platform-forked";
```

Comment:
Have you opened a PR to upstream your changes? Or could you leave a comment on the nature of your changes?